merge soberdocs into vinegarhq.org (#143)
vinegarhq/vinegarhq.github.io: src/Sober/FAQ/index.md

# Sober FAQ

## Q: Why Does this exist?
A: Because people on Linux still want to be able to play Roblox easily.

## Q: What is the minimum hardware needed?
In order to run Sober you need:

> * CPU: Any x86_64 CPU with SSE4.2 or higher (CPUs from 2013+ should be fine)

Reviewer: Maybe we could add a footnote explaining how to check CPU capabilities?

Author: Example?

Reviewer 2: Having the user run something like `grep -q "sse4_2" /proc/cpuinfo && echo available` would work. There might be a better way to do it though.

(could always just `grep "sse4_2" /proc/cpuinfo` too, but i personally would do the other one)

Author: Added the command in FAQ

* RAM: 4 GB, although it might run with less
* GPU: Vulkan capable graphics card, check "Q: Is the [GPU here] compatible?" for more info*
* Storage: 1-2 GB

> <p class="tiny">[*] Vulkan 1.3 is required for "Future" graphics.<p>

Reviewer: Doesn't Sober have an OpenGL mode?

Author: Oh yeah, I forgot about adding OpenGL reqs in here

![Vulkan Supported](./vulkaninfo.png)

## Q: I can't log in to Roblox!
A: If it didn't say explicitly you have a wrong password or something like that, reopen Sober and check if it logged you in. If that didn't work, try **"Login with Another Device"** on the login page.

## Q: Studio support?
A: Roblox does not have Studio for Android. Use [Vinegar](https://vinegarhq.org/) for Roblox Studio, which uses the Windows version of Roblox Studio with Wine.

## Q: Why is Sober closed source?
A: When the development team was making Sober, they wanted to ensure it wouldn't meet the same fate as Vinegar (being blocked). They recommended to make Sober closed source, to ensure no Exploit developers abuse it. Sadly, this is a necessary measure to ensure the continued existence of Roblox on Linux. On the bright side, this ensures that Sober will not be blocked in the long term, if no exploits are made for it, which the dev team will ensure.

> ### SQ: But I heard Roblox is working on Hyperion for Android
> A: There is, and it's being worked on as we speak. And while progress towards activation seems to be slow, Hyperion on Android is most likely going to release this year, possibly within Q1 of this year. Sober is most unlikely to be continued once Hyperion has been deployed.

Reviewer: ```suggestion
### SQ: Wasn't Roblox working on Hyperion for Android? What happens then?
A: That's correct. Despite progress appearing to be slow, it's very likely that Hyperion gets deployed on Android during 2025, possibly within Q1. Once this happen, Sober will probably be discontinued.
```

## Q: Roblox is crashing after around 5-10 minutes in game!
A: There are various issues that could cause this. (You can check [the list of common issues](../Troubleshooting/index.md#known-general-issues)) Mostly, if you're on Wayland it's because Wayland support isn't great at the moment so you'll want make Sober use XWayland. To do this, you can simply run ```flatpak override --user --socket=x11 --nosocket=wayland org.vinegarhq.Sober```. Alternatively, if you prefer a GUI for managing flatpak permissions, you can use [Flatseal](https://flathub.org/apps/com.github.tchx84.Flatseal). And if you're on X11, don't know how to help you.

![Turn on XWayland](./xwaylandforever.png)

> ## Q: ARM64 support?
> A: Maybe eventually, but right now there are too many issues to deal with to add support right now.

Reviewer: ```suggestion
## Q: ARM64 support?
A: We currently have no plans to support ARM64. It'd take a huge amount of time and effort we can't spare. However, this isn't definitive and could be reconsidered in the future.
```

# Sober FAQ

## Q: Why Does this exist?
A: Because people on Linux still want to be able to play Roblox easily.

## Q: How do I know I meet the requirements to run Sober?
- If you know your CPU very well, any CPUs that is from 2008+ for Intel (Nehalem series or later) or 2011+ for AMD (Bulldozer series or later) should be fine.

>     - If you don't know if your CPU supports SSE4.2, you can run `rep "sse4_2" /proc/cpuinfo` on your terminal. If SSE4.2 is available, it should be highlighted when you run the command.

Reviewer 2: `grep` is misspelled as `rep`

```suggestion
    - If you don't know if your CPU supports SSE4.2, you can run `grep "sse4_2" /proc/cpuinfo` on your terminal. If SSE4.2 is available, it should be highlighted when you run the command.
```

Author: oh I thought I copy and pasted correctly whoops

- a Sober dev
```

## Q: Roblox is crashing after around 5-10 minutes in game!
A: There are various issues that could cause this. (You can check [the list of common issues](../Troubleshooting/index.md#known-general-issues)) Mostly, if you're on Wayland it's because Wayland support isn't great at the moment so you'll want make Sober use XWayland. To do this, you can simply run ```flatpak override --user --socket=x11 --nosocket=wayland org.vinegarhq.Sober```. Alternatively, if you prefer a GUI for managing flatpak permissions, you can use [Flatseal](https://flathub.org/apps/com.github.tchx84.Flatseal). And if you're on X11, don't know how to help you.

![Turn on XWayland](./xwaylandforever.png)

## Q: ARM64 support?
A: Maybe eventually, but right now there are too many issues to deal with to add support right now.

## Q: Will Sober be released on Flathub?
A:
```
sober is experimental, and putting it on flathub implies that it's ready for production use, and that it's supported software that is likely to keep working
sober isn't ready for production use, it's not supported, and it can stop working at any time, for any reason

- a Sober dev

> ```

Reviewer: Maybe we should use a quote for this, instead of a codeblock?

Reviewer 3: Or alternatively paraphrase, keeping this grammar looks inconsistent with the rest of the documentation.

## Q: Does it work with VR?
A: No, as the Android version doesn't have support for VR. The Quest version is only on built on ARM and under a translation layer, it performs horribly.

> 

Reviewer: Kids these days don't know how to troubleshoot anymore. :P 

Maybe we should add one of those "please just try again" hints?
```suggestion

## Q: I'm running into an issue which isn't listed here. What should I do?
A: More often than not, simply restarting Sober or updating it to the latest version fixes the issue. If it persists, consider reporting it in VinegarHQ's Discord Server.
```
